v3/lints/cabf_cs_br/lint_cs_rsa_key_size.go

```diff
@@ -0,0 +1,58 @@
+package cabf_cs_br
+
+import (
+	"crypto/rsa"
+
+	"github.com/zmap/zcrypto/x509"
+
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/util"
+)
+
+/*6.1.5.2 Code signing Certificate and Timestamp Authority key sizes
+For Keys corresponding to Subscriber code signing and Timestamp Authority Certificates:
+• If the Key is RSA, then the modulus MUST be at least 3072 bits in length.
+• If the Key is ECDSA, then the curve MUST be one of NIST P‐256, P‐384, or P‐521.
+• If the Key is DSA, then one of the following key parameter options MUST be used:
+• Key length (L) of 2048 bits and modulus length (N) of 224 bits
+• Key length (L) of 2048 bits and modulus length (N) of 256 bits*/
+
+func init() {
+	lint.RegisterCertificateLint(&lint.CertificateLint{
+		LintMetadata: lint.LintMetadata{
+			Name:          "e_cs_rsa_key_size",
+			Description:   "If the Key is RSA, then the modulus MUST be at least 3072 bits in length",
```

Comment:
I agree that each key type is likely deserving of their own lint. Requirements on specific algorithms are usually best left separated.

Reply:
This is the main difference between RSA key size requirements between CS and TLS. Other types like ECDSA are covered in other lints.